tools/dashboard-plugin-template/webpack.config.js

```diff
@@ -143,7 +143,7 @@ module.exports = (_env, argv) => {
                     PORT: JSON.stringify(PORT),
                 }),
                 new Dotenv({
-                    silent: true, // we are ok with the .env file not being there, do not warn about it (so that CI works)
+                    silent: false, // we are ok with the .env file not being there, do not warn about it (so that CI works)
```

Comment:
The comment is no longer valid. But I suspect this is what is nuking your build currently.

Reply:
interesting, i don't even remember turning it to false :) anyway, i've made it silent & have two entries one for default .env and one for .env.secrets